### PR TITLE
Moving __future__ import to top of file

### PR DESCRIPTION
**Pull request by bsb808**
Should fix the bug discovered when running the example for Dave PR: https://github.com/Field-Robotics-Lab/dave/pull/149

If the __future__ import is not at the beginning of the file, a SyntaxError is raised:
```
$ ./gripper_controller.py 
Traceback (most recent call last):
  File "./gripper_controller.py", line 18, in <module>
    from uuv_manipulator_interfaces import GripperInterface
  File "/home/bsb/uuv_ws/devel/lib/python3/dist-packages/uuv_manipulator_interfaces/__init__.py", line 34, in <module>
    exec(__fh.read())
  File "<string>", line 17, in <module>
  File "/home/bsb/uuv_ws/src/uuv_manipulators/uuv_manipulators_kinematics/src/uuv_manipulator_interfaces/gripper.py", line 25
    from __future__ import division
    ^
SyntaxError: from __future__ imports must occur at the beginning of the file
```

### CODE REVIEW

**Comment by M1chaelM** [comment]
This PR is a duplicate of #4 (contributed by a community member), so we should close that one after merging this. I added another small tweak to update to Python3 syntax, just in the interest of reducing the total number of PRs.

File: uuv_manipulators_kinematics/src/uuv_manipulator_interfaces/gripper.py [hunk 1]
```diff
@@ -22,7 +24,10 @@
 from std_msgs.msg import Float64
 from sensor_msgs.msg import JointState
 import time
-from __future__ import division
+<<<<<<< Updated upstream
+
+=======
+>>>>>>> Stashed changes
```
Comment (mabelzhang) [inline uuv_manipulators_kinematics/src/uuv_manipulator_interfaces/gripper.py]:
```suggestion
```

Comment (M1chaelM) [reply uuv_manipulators_kinematics/src/uuv_manipulator_interfaces/gripper.py]: Oops! Thank you.

**Review by mabelzhang — APPROVED** [review]
This works for me! Thank you!

**Comment by mabelzhang** [comment]
Extra steps, but if we want to keep the earlier contribution from #4, we could merge #4, merge master into this PR, and then merge this.

**Pull request merged**